src/engraving/types/accessibilitystyle.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-Studio-CLA-applies
+ *
+ * MuseScore Studio
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#ifndef MU_ENGRAVING_ACCESSIBILITYSTYLE_H
+#define MU_ENGRAVING_ACCESSIBILITYSTYLE_H
+
+namespace mu::inspector {
```

Comment:
This uses the `inspector` namespace, but is in the `engraving` module. Did you mean to put it in the `inspector` module?

Reply:
I think that is a typo - should be in the `engraving` module only. Will fix this

Reply:
But the only place where it is used is the inspector module, so are you sure about that?

Reply:
Also, if you will only need to use it in C++, then there is probably no need to create an `AccessibilityStyleTypes` wrapper class. Instead, you could just create an `AccessibilityStylePreset` enum, not wrapped in a class.

Reply:
The idea is as the `preset` attribute needs to go into the `Style` tag, I was thinking the time when a user selects a preset value, it should set a member variable of the `Style` class say `m_stylepreset` according to the preset value selected so the method could look like:
```cpp
void MStyle::setStylePreset(AccessibilityStyleTypes::StylePreset presetName)
{
    switch(presetName) {
    case AccessibilityStyleTypes::StylePreset::MSN_16MM:
        m_stylepreset = "16mm MSN";
    case AccessibilityStyleTypes::StylePreset::MSN_18MM:
        m_stylepreset = "18mm MSN";
    case AccessibilityStyleTypes::StylePreset::MSN_20MM:
        m_stylepreset = "20mm MSN";
    case AccessibilityStyleTypes::StylePreset::MSN_22MM:
        m_stylepreset = "22mm MSN";
    case AccessibilityStyleTypes::StylePreset::MSN_25MM:
        m_stylepreset = "25mm MSN";
    default:
        return "";
    }
}
```
and this value could be used in the .mss for the `Style` tag

Reply:
A class is a real physical structure in memory. You shouldn't use it just as a namespace.

I think this will be fine:

```C++
void MStyle::setStylePreset(AccessibilityStylePreset presetName)
```

So no need for the `AccessibilityStyleTypes` class.